src/DotNetWorker.Grpc/GrpcServiceCollectionExtensions.cs

```diff
@@ -68,9 +67,30 @@ public static IServiceCollection AddGrpc(this IServiceCollection services)
 #endif
 
             services.AddOptions<GrpcWorkerStartupOptions>()
-                .Configure<IConfiguration>((arguments, config) =>
+                .Configure<IConfiguration>((grpcWorkerStartupOption, config) =>
                 {
-                    config.Bind(arguments);
+                    Uri? grpcUri;
+                    var functionsUri = config["functions-uri"];
```

Comment:
Let's avoid the code duplication here and reuse the logic used int he `Worker.Extensions.Grpc` project

Reply:
This is still not addressed based on the [latest comment ](https://github.com/Azure/azure-functions-dotnet-worker/pull/1897#discussion_r1326259085)from @jviau